src/Admin/ReaderThemes.php

```diff
@@ -210,17 +227,45 @@ public function get_default_reader_themes() {
 				]
 			);
 
-			if ( ! is_wp_error( $response ) ) {
-				set_transient( $cache_key, $response, DAY_IN_SECONDS );
+			if ( is_array( $response ) ) {
+				$response = (object) $response;
 			}
-		}
 
-		if ( is_wp_error( $response ) ) {
-			return [];
-		}
+			/**
+			 * The response must minimally be an object with a themes array.
+			 *
+			 * @see https://wordpress.org/support/topic/issue-during-activating-the-updated-plugins/#post-13383737
+			 */
+			if (
+				! is_object( $response )
+				|| ! property_exists( $response, 'themes' )
+				|| ! is_array( $response->themes )
+				|| is_wp_error( $response )
+			) {
+				if ( is_wp_error( $response ) ) {
+					$message = $response->get_error_message();
+				} else {
+					$message = __( 'The request for reader themes from the WordPress.org resulted in an invalid response. Check your Site Health to confirm that your site can communicate with WordPress.org. Otherwise, please try again later or contact your host.', 'amp' );
+				}
```

Comment:
```suggestion
				$message = __( 'The request for reader themes from the WordPress.org resulted in an invalid response. Check your Site Health to confirm that your site can communicate with WordPress.org. Otherwise, please try again later or contact your host.', 'amp' );
				if ( is_wp_error( $response ) && defined( 'WP_DEBUG_DISPLAY' ) && WP_DEBUG_DISPLAY ) ) {
					$message .= ' ' . __( 'Error message:', 'amp' ) . ' ' . $response->get_error_message();
				}
```

Reply:
Ah, I see what you mean. Thanks!